examples/inference_restoring_model_dependencies/sklearn_model_inference_restoring_dependencies.py

```diff
@@ -0,0 +1,26 @@
+import mlflow
+import sklearn.datasets as datasets
+from sklearn.neighbors import KNeighborsClassifier
+import tempfile
+from pyspark.sql import SparkSession
+
+
+spark = SparkSession.builder.getOrCreate()
+
+
+X, y = datasets.load_iris(as_frame=True, return_X_y=True)
+model = KNeighborsClassifier()
+model.fit(X, y)
+
+model_path = tempfile.mkdtemp()
+
+mlflow.sklearn.save_model(model, model_path)
```

Comment:
```suggestion
mlflow.sklearn.log_model(model, model_path)
```

Can we use `log_model`?